models/articles.models.js

```
@@ -23,4 +23,20 @@ exports.getAllArticlesData = () => {
         GROUP BY articles.article_id
         ORDER BY articles.created_at DESC
         `);
+}
+
+exports.getCommentsByArticleIdData = (article_id) => {
+    if (/^\d+$/.test(article_id)) {
```

If the article_id is an invalid type then PSQL should throw an error